### PR TITLE
Fixed link and formatting issue in '/openstack/install"

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates links to point to /openstack/tutorials
- Updates to correct asset and fixes formatting

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/install
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare against [copydoc](https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc/edit)

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/openstack/install.html [hunk 1]
```diff
@@ -166,21 +166,29 @@ <h2>Large-scale deployment</h2>
...
+    <div class="col-7 u-vertically-center">
       <h2>OpenStack tutorials</h2>
       <p>Learn OpenStack through a series of tutorials.</p>
-      <p><a class="p-button--positive js-invoke-modal" href="/openstack/tutorials">Explore tutorials</a></p>
+      <p><a class="p-button--positive" href="/openstack/tutorials">Explore tutorials</a></p>
     </div>
   </div>
 </section>
 
 <section class="p-strip--light">
```
Comment (minkyngkm) [inline templates/openstack/install.html:189]:
```suggestion
<section class="p-strip">
```
Would this be better as it's new section?

**Review by minkyngkm — APPROVED** [review]
LGTM! 👍 just left one suggestion if it's better :)

**Comment by minkyngkm** [comment]
in line 157, there seems to be no space between "of" and "OpenStack". It's not from your update but it would be good if it can be updated! :) 

image

File: templates/openstack/install.html (hunk 1 above)
Comment (petesfrench) [reply templates/openstack/install.html:189]: Good suggestion! I also added a border on the bottom to break the up further.

**Pull request merged**